.github/dependabot.yml

```diff
@@ -22,3 +22,10 @@ updates:
       day: tuesday
     commit-message:
       prefix: "chore"
+  - package-ecosystem: npm
```

Comment:
Important: Set versions 1 version to low so post merge we can validate dependabot updates.